docs/rules/no-node-access.md

```diff
@@ -51,6 +51,25 @@ within(signinModal).getByPlaceholderText('Username');
 document.getElementById('submit-btn').closest('button');
 ```
 
+## Options
+
+This rule has one option:
+
+- `allowContainerFirstChild`: **disabled by default**. When we have container
+  with rendered content then the easiest way to access content itself is by using
+  `firstChild` property. Use this option in cases when this is hardly avoidable.
```

Comment:
Could you add a link to the Testing Library docs where this is recommended? https://testing-library.com/docs/react-testing-library/api/#container-1

Reply:
Added